README.md

```diff
@@ -366,6 +366,7 @@ Check out the following links to get a better idea of what to expect:
 | Design a deck of cards | [Solution](solutions/object_oriented_design/deck_of_cards/deck_of_cards.ipynb)  |
 | Design a parking lot | [Solution](solutions/object_oriented_design/parking_lot/parking_lot.ipynb)  |
 | Design a chat server | [Solution](solutions/object_oriented_design/online_chat/online_chat.ipynb)  |
+| Design a cab booking | [Solution](https://youtu.be/Yn7C0x5ozx4)  |
```

Comment:
A YouTube channel, is it allowed ??

Reply:
I think so. If you want, I can change it to github repo also: https://github.com/anomaly2104/lld-cab-booking-ola-uber-grab-lyft